src/commands/setup.rs

```diff
@@ -31,7 +31,12 @@ impl Setup {
 
         let network_options = match network::types::NetworkOptions::load(&input_file) {
             Ok(opts) => opts,
-            Err(e) => panic!("{}", e),
+            Err(_) => {
```

Comment:
I think `e` here should be passed into the `error` message instead of ignoring by (`_`). We could use something like `format!("Netvark error {}", e).to_string()` or something better. But overall I think we can't ignore original error.

Reply:
ack

Reply:
done